linux_sql/sql/ddl.sql

```diff
@@ -0,0 +1,32 @@
+DROP TABLE host_info, host_usage;
```

Comment:
The database is not mentioned in this file.
The file should contain commands to create a database & connect to it. 
PUBLIC prefix would not be needed.